### ISSUE TITLE
Some nil error

### ISSUE THREAD

**Issue by johnnybgood89** [obble/modui#89]
Hello, 
I got some nil error, here is the scr shot

http://imgur.com/MAd158q

**Comment by Road-block** [comment]
Another addon is probably loading another HealComm instance early or you have an outdated version of HealComm standalone in your AddOns folder.  
Normally libraries should have an incremental release number that they check during the loading process discarding older versions found but I doubt library developers / backporters for vanilla take the time to check that nowadays (It's also very hard with multiple versions of addons in the wild)  

**TL;DR;** Search in your addons folder for `HealComm` potentially update other addons (or the library itself if you have it installed standalone) or manually copy the newest version over the others.

**Comment by johnnybgood89** [comment]
I dont have addon HealComm or any heal addons :/ 

Ok Ill check

**Comment by obble** [comment]
yeah please check that you are using the latest version of modui and any other major addons you're loading.

healcomm is often embedded in addons such a LunaUnitframes

**Comment by johnnybgood89** [comment]
I have that but think Im not using it at all. I ll delete it and check. Thanks
